src/components/presence-mouse/canvas/index.ts

```diff
@@ -32,6 +36,9 @@ export class PointersCanvas extends BaseComponent {
     this.animateFrame = requestAnimationFrame(this.animate);
 
     this.goToMouseCallback = options?.onGoToPresence;
+
+    const { localParticipant } = this.useStore(StoreType.GLOBAL);
+    localParticipant.subscribe();
```

Comment:
You don't need to pass a callback to reassign the local participant when it's updated?

Reply:
Thanks to what happens when you call this.useStore(), no. You only need to pass a callback when you don't want to reassign a property with that exact name. For example, if you wanted to extract the local participant name in a property every time instead of the entire participant, you would need a callback to specify this behavior. Without a callback, `this.localParticipant = localParticipant` already happens when someone updates the localParticipant, for example.